src/BlogBundle/Posts/2020-05-28-announcing-rawcooked-21-01.md

```diff
@@ -0,0 +1,47 @@
+---
+title:  "Announcing RAWcooked 21.01!"
+date:   2021-01-04 08:00:00 CET
+excerpt: "Thanks to support from New York Public Library, MediaArea is proud to present a new and improved version of RAWcooked"
+tags: RAWcooked
+---
+
+Announcing RAWcooked 21.01!
+
+We're very grateful to announce that a new version of RAWcooked is now available. The 21.01 release introduces several new features and fixes that focus on ensuring that the full reversibility of RAWcooked files to their source files is verifiable, even if the source files are not later accessible. Often users were converting DPX to RAWcooked and then back to DPX to test the reversibility as a multi-step process and now those tests, along with many other conformance tests, are included as part of the RAWcooked process. We recommend updating to this version and offering any feedback directly to us or via GitHub.

```

Comment:
Is “grateful” correct? (I’m not English speaker, but to me it sounds strange.)

Reply:
> Is “grateful” correct? 

[yes](https://www.linguee.fr/anglais-francais/traduction/we+are+very+grateful.html).

Reply:
Through introspection, I've found that I do feel a quality of being thankful for the opportunity to contribute to and to announce this new release. The gratitude that I feel comes from the support, collaboration, and problem-solving of users and developers to create such a work in response to identified needs and challenges in film preservation, which is a personal topic for me. I consider delving too far into personal psychology may be out of scope of the pull request, but I confirm that I do feel grateful (not speaking for @JeromeMartinez).